tests/www/views/test_views_base.py

```diff
@@ -68,7 +68,7 @@ def heartbeat_healthy():
         state="running",
         latest_heartbeat=last_heartbeat,
     )
-    SchedulerJobRunner(job=job),
+    (SchedulerJobRunner(job=job),)
```

Comment:
Seems like it would be easier to found extra comma in the end, which turned into tuple.
There is no critical here because we do not assign anything, but better to fix it

```suggestion
    SchedulerJobRunner(job=job)
```